src/breadcrumb/breadcrumb.stories.ts

```diff
@@ -0,0 +1,59 @@
+import { storiesOf, moduleMetadata } from "@storybook/angular";
+import { withNotes } from "@storybook/addon-notes";
+import { action } from "@storybook/addon-actions";
+import { withKnobs, boolean, number } from "@storybook/addon-knobs/angular";
+
+import { BreadcrumbModule, DialogModule } from "../";
+
+const MAXIMUM_ITEM_COUNT = 50;
+
+storiesOf("Breadcrumb", module)
+	.addDecorator(
+		moduleMetadata({
+			imports: [BreadcrumbModule, DialogModule]
+		})
+	)
+	.addDecorator(withKnobs)
+	.add("Basic", () => ({
+		template: `
+		<ibm-breadcrumb [noTrailingSlash]="noTrailingSlash">
+			<ibm-breadcrumb-item href="#1">
+				Breadcrumb 1
+			</ibm-breadcrumb-item>
+			<ibm-breadcrumb-item href="#2">
+				Breadcrumb 2
+			</ibm-breadcrumb-item>
+			<ibm-breadcrumb-item href="#3">
+				Breadcrumb 3
+			</ibm-breadcrumb-item>
+			<ibm-breadcrumb-item href="#4">
+				Breadcrumb 4
+			</ibm-breadcrumb-item>
+		</ibm-breadcrumb>`,
+		props: {
+			noTrailingSlash: boolean("noTrailingSlash", true)
```

Comment:
What do you think of having a `number` prop/knob which would allow changing item count in breadcrumb component?

That way users would be able to see how the count affect the visuals.

```html
<ibm-breadcrumb [noTrailingSlash]="noTrailingSlash">
	<ibm-breadcrumb-item *ngFor="let i of range(itemCount)" href="#{{i + 1}}">
		Breadcrumb {{i + 1}}
	</ibm-breadcrumb-item>
</ibm-breadcrumb>
```

_(`range` is available in `common/utils`)_

We could still keep both stories for convenience but provide knobs for both.

Reply:
@zvonimirfras I have tried that but cannot get that working. I met with @cal-smith and we decided to make this a follow up item, see issue #130